parse.py

```diff
@@ -1053,13 +1060,21 @@ def watch_keyboard(keyboard, use_hotkeys):
     keyboard.clipboard_callback = lambda _: hotkey_handler(keyboard, "clipboard")
```

Comment:
Clipboard_callback is not used as well (and ctrl+c hotkey still missing)

Reply:
Also, the application should handle scenario where both pynput and keyboard are not working, and clipboardwatcher was handling that at least for copy-paste